pyaedt/generic/general_methods.py

```diff
@@ -1017,6 +1017,67 @@ def grpc_active_sessions(version=None, student_version=False, non_graphical=Fals
     return sessions
 
 
+def active_sessions(version=None, student_version=False, non_graphical=False):
+    """Get information for the active COM AEDT sessions.
+
+    Parameters
+    ----------
+    version : str, optional
+        Version to check. The default is ``None``, in which case all versions are checked.
+        When specifying a version, you can use a three-digit format like ``"222"`` or a
+        five-digit format like ``"2022.2"``.
+    student_version : bool, optional
+    non_graphical : bool, optional
+
+
+    Returns
+    -------
+    list
+        List of AEDT PIDs.
```

Comment:
Am I the only one that doesn't know what a PID is?

Reply:
@PipKat It stands for process ID.